File: cSharp/day52.cs
using System;
using System.Text.RegularExpressions;

Console.WriteLine("Please input your desired password");
string password = Console.ReadLine();


Regex hasDigits = new Regex("[A-Z]");


if (hasDigits.Match(password).Success) {
    Console.WriteLine("Password is valid");
}
else {
    Console.WriteLine("Password is INVALID");
}

File: go/day52.go
package main

import (
	"bufio"
	"fmt"
	"os"
	"strings"
	"regexp"
)

func main() {
	fmt.Println("Please input your password")

	reader := bufio.NewReader(os.Stdin)

	password1,_ := reader.ReadString('\n')
	password1 = strings.TrimSpace(password1)

	match, _ := regexp.MatchString("[A-Z]", password1)

	fmt.Println("Valid password: ", match)
}